docs/src/main/tut/datatypes/either.md

```diff
@@ -212,132 +212,6 @@ magic("123") match {
 }
 ```
 
-## Either in the small, Either in the large
-Once you start using `Either` for all your error-handling, you may quickly run into an issue where
```

Comment:
Nonetheless, the whole section was presented as a 'solution' - but the problem being solved does not exist anymore. To quote the relevant part: 'If you're on Scala 2.12, this line will compile and work as expected (...)'. Everything that followed seemed to have been written as a guide to deal with issues on previous versions of Scala. It of course does not invalidate ideas contained therein. Obviously, organizing your errors as ADTs is a valid and useful technique. But a rather generic one and not really tied to usage of `Either` in cats. Of course I'm open to any suggestions, but I must admit that I simply *do not know* what should be there instead.